### PR TITLE
Fixed inital unit

### PR DESCRIPTION
**Pull request by karthshen**

### CODE REVIEW

File: src/main/game/GameFailedToStartException.java [hunk 1]
```diff
@@ -0,0 +1,4 @@
+package game;
+public class GameFailedToStartException extends Exception {
+    public GameFailedToStartException(){ super("Game failed to start.\n"); }
```
Comment (gdscheele) [inline src/main/game/GameFailedToStartException.java]: I think it automatically puts a newline

File: src/main/game/GameModel.java [hunk 2]
```diff
@@ -23,26 +34,50 @@
...
+            gameHasStarted = true;
+            startTurn();
+        }catch(GameFailedToStartException e){
+            System.out.println(e.getMessage());
+        }
+
     }
     
     public void updateGame() { //This is called up to 60 times per second
     	if (gameHasStarted) {
```
Comment (gdscheele) [inline src/main/game/GameModel.java]: I know you didn't just add this, but we probably don't want to check if the game is over 60 times a second, just after each turn/move

File: src/main/game/GameModel.java [hunk 3]
```diff
@@ -23,26 +34,50 @@
...
     		checkIfGameOver();
     	}
     }
+
+    public void initialUnit(Player human, Player panda) throws GameFailedToStartException {
+        try {
+            UnitFactory unitFactory = new UnitFactory();
+
+            Colonist humanColnist = (Colonist)unitFactory.createUnit(EntitySubtypeEnum.COLONIST,HUMAN_STARTING_LOCATION, 0);
```
Comment (gdscheele) [inline src/main/game/GameModel.java]: spelling

File: src/main/game/GameModel.java [hunk 4]
```diff
@@ -23,26 +34,50 @@
...
+
     }
     
     public void updateGame() { //This is called up to 60 times per second
     	if (gameHasStarted) {
     		checkIfGameOver();
     	}
     }
+
+    public void initialUnit(Player human, Player panda) throws GameFailedToStartException {
```
Comment (gdscheele) [inline src/main/game/GameModel.java]: can you change this to initializeUnits? or addInitialUnitsToGameboard? something like that

File: src/main/game/GameModel.java (hunk 2 above)
Comment (karthshen) [inline src/main/game/GameModel.java]: Are you talking about the updateGame() or initializeGame()?

Comment (gdscheele) [inline src/main/game/GameModel.java]: checkIfGameOver()

**Pull request merged**